packages/tools/devtools/devtools-core/src/FluidDevtools.ts

```diff
@@ -134,6 +138,15 @@ export class FluidDevtools implements IFluidDevtools {
 	 */
 	private readonly containers: Map<ContainerKey, ContainerDevtools>;
 
+	/**
+	 * Stores DataObject-level devtools instances registered with this object.
+	 * Maps from a {@link ContainerKey} to the corresponding {@link DataObjectDevtools} instance.
+	 */
+	private readonly dataObjects: Map<ContainerKey, DataObjectDevtools>;
+
+	// Track data object instances to assign sequential numbers
```

Comment:
Let's make this a TSDoc-style comment. Also probably merits a bit more detail explaining how this is used.